src/widget/wmainmenubar.cpp

```diff
@@ -300,6 +300,22 @@ void WMainMenuBar::initialize() {
             pViewFullScreen, SLOT(setChecked(bool)));
     pViewMenu->addAction(pViewFullScreen);
 
+    QString toggleMenuBarTitle = tr("&Hide menu bar");
+    QString toggleMenuBarText = tr("Hide menu bar");
+    QAction* pToggleMenuBar = new QAction(toggleMenuBarTitle, this);
+    pToggleMenuBar->setShortcut(
+        QKeySequence(m_pKbdConfig->getValueString(ConfigKey("[KeyboardShortcuts]",
+                                                  "ViewMenu_ToggleMenuBar"),
+                                                  "F12")));
```

Comment:
It is common in other Applications like Firefox or Nemo to show the menu bar if the ALT key is pressed.
Mixxx should do the same.